src/components/PeopleData.tsx

```diff
@@ -0,0 +1,60 @@
+const people = [
```

Comment:
We want to have this as a typed object:
```
interface Person {
  readonly name: string;
  readonly position: string;
  ....
}
```

Then have: 
`const people: People[] = [...`